content/core-features/occupancy.textile

```diff
@@ -2,16 +2,16 @@
 
-The metrics returned by the occupancy feature include the number of clients connected to a channel, the number of those authorized to subscribe to messages on the channel and the number of those clients authorized to publish to the channel. The occupancy metrics also include the number of connections authorized to subscribe to presence events, the number authorized to publish presence events and the number of connections currently entered into "presence":/core-features/presence on the channel.
+The metrics returned by the occupancy feature include the number of clients attached to a channel, the number of those authorized to subscribe to messages on the channel and the number of those clients authorized to publish to the channel. The occupancy metrics also include the number of connections authorized to subscribe to presence events, the number authorized to publish presence events and the number of "presence":/core-features/presence members currently present on the channel.
 
-Channel occupancy can be queried in a number of ways. Occupancy can be viewed as part of "channel enumeration":/rest-api#enumeration-rest, configured as an "integration rule source":/general/events#sources or as "inband channel occupancy events":/realtime/inband-occupancy. 
+Channel occupancy can be queried in a number of ways. Occupancy can be viewed as part of the "channel status API":/rest/channel-status, when configured as a "@channel.occupancy@ rule source":/general/events#sources or as "inband channel occupancy events":/realtime/inband-occupancy. 
 
 h2(#occupancy-presence). Occupancy and presence
 
-Occupancy differs from "presence":/core-features/presence in that presence enables a client to identify themselves to other clients subscribed to presence events on a channel, as well as to update their member data for subscribers to view. Presence events are emitted when a client enters or leaves a channel or when they update their member data. Occupancy provides a count of the clients connected to a channel, broken down into specific metric categories. Occupancy data can be viewed at a specific point in time when using channel enumeration, or emitted as events when metrics change using an integration rule source or inband channel occupancy.
+Occupancy differs from "presence":/core-features/presence in that presence enables a client to identify themselves to other clients subscribed to presence events on a channel, as well as to update their member data for subscribers to view. Presence events are emitted when a client enters or leaves a channel or when they update their member data. Occupancy provides a count of the clients attached to a channel, broken down into specific metric categories. Occupancy data can be viewed at a specific point in time when using the channel status API, or emitted as events when metrics change using a @channel.occupancy@ rule source or inband channel occupancy events.
```

Comment:
This all seems a bit wordy. Perhaps something like the following?

The occupancy and presence features differ in the following ways:

**Occupancy**: Provides information about the _number of members in the channel_. This includes a count of the number of clients attached to the channel, the number of publishers and subscribers, and how many clients subscribe to presence events.

**Presence**: Provides information about the _activity of individual channel members_. Clients attached to a channel announce their state to other clients in the channel via presence events. For example, when entering a channel, leaving a channel, or updating their member data. Other clients can subscribe to these events.